static/src/javascripts-legacy/projects/common/modules/experiments/tests/membership-engagement-banner-tests.js

```diff
@@ -27,7 +27,7 @@ define([
         this.author = 'Roberto Tyley';
         this.description = 'Show contributions/membership messages for the ' + edition + ' edition.';
         this.showForSensitive = false;
-        this.audience = 0.5;
+        this.audience = 1.0;
```

Comment:
Watch out. Changing the audience of a running test will have consequences for the data already collected.